### ISSUE TITLE
Are the wrong classified images sorted out? 

### ISSUE THREAD

**Issue by jS5t3r** [bethgelab/foolbox#713]
Assume a pre-trained classifier.
The clean accuracy is 87.5 %. 

There 12.5% which are wrongly classified on a pre-trained PyTorch model before running the attack. 
What happens with these 12.5% within the `attack` method? 

```python
xp_, x_, success = attack(fmodel, images, labels, epsilons=[8./255.])
```

Are these 12.5% ignored in advance?

**Comment by EatonL** [comment]
It should have overlooked these samples. I input a total of 16 images and output, and there are only 13 clipped_advs samples, and here is the process of my attack. I don't know why there are 13 samples.

  Linf norm ≤ 0.0   : 93.8 %
  Linf norm ≤ 0.0002: 87.5 %
  Linf norm ≤ 0.0005: 81.2 %
  Linf norm ≤ 0.0008: 50.0 %
  Linf norm ≤ 0.001 : 37.5 %
  Linf norm ≤ 0.0015: 12.5 %
  Linf norm ≤ 0.002 :  6.2 %
  Linf norm ≤ 0.003 :  0.0 %
  Linf norm ≤ 0.01  :  0.0 %
  Linf norm ≤ 0.1   :  0.0 %
  Linf norm ≤ 0.3   :  0.0 %
  Linf norm ≤ 0.5   :  0.0 %
  Linf norm ≤ 1.0   :  0.0 %

**Comment by zimmerrol** [comment]
No, they are not ignored. Why is this the case? The criterion for when a sample should be ignored depends on the attack: if you run an untargeted attack the 12.5% of samples should be ignored; for an untargeted attack, however, it doesn't make sense to ignore them.
